vaadin-spring/src/main/java/com/vaadin/flow/spring/VaadinServletContextInitializer.java

```diff
@@ -520,12 +522,19 @@ public void contextDestroyed(ServletContextEvent sce) {
      * {@code context} provided.
      *
      * @param context
-     *            the application context
+     *                the application context
      */
     public VaadinServletContextInitializer(ApplicationContext context) {
         appContext = context;
         String neverScanProperty = appContext.getEnvironment()
-                .getProperty("vaadin.blacklisted-packages");
+                .getProperty("vaadin.blocked-packages");
+        if (neverScanProperty == null) {
+            neverScanProperty = appContext.getEnvironment()
+                    .getProperty("vaadin.blacklisted-packages");
+            if (neverScanProperty != null) {
+                getLogger().warn("vaadin.blacklisted-packages is deprecatd. Use vaadin.blocked-packages instead.");
```

Comment:
How about putting a little more pressure on the developer?

```suggestion
                getLogger().warn("vaadin.blacklisted-packages is deprecatd and may not be supported anymore in the future. Use vaadin.blocked-packages instead.");
```

Reply:
Sure, thanks. Fixed one typo at the same time.